core/src/main/java/com/hcl/ecomm/core/servlets/ProductDetailsServlet.java

```diff
@@ -40,19 +40,19 @@ protected void doGet(SlingHttpServletRequest request, SlingHttpServletResponse r
 		String  sku = request.getParameter("sku");
         if(StringUtils.isNotEmpty(sku)) {
 		
-        JsonObject productResponse = productService.getProductDetail(sku);
```

Comment:
@Subrat-Dash-416 Why this file change in junit ?

Reply:
same as above